app/assets/stylesheets/homepage.css.scss

```diff
@@ -26,7 +26,7 @@
 
 #logo {
     text-decoration: none;
-    margin-left: -70px ;
+    margin-left: 0px ;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Declaration should not have a space before the terminating semicolon<br>`0px` should be written without units as `0`